spark/src/main/scala/geotrellis/spark/io/package.scala

```diff
@@ -50,6 +52,25 @@ package object io
         layoutRows = that.segmentLayout.tileLayout.layoutRows)
   }
 
+  private[io] def updateQueriedMetadata[
+    K: SpatialComponent: Boundable,
+    M: Component[?, Bounds[K]]: Component[?, LayoutDefinition]: Component[?, Extent]
+  ](
+    keyBounds: Seq[KeyBounds[K]],
+    sourceMetadata: M
+  ): M = {
+    val mapTransform = sourceMetadata.getComponent[LayoutDefinition].mapTransform
+
+    if (keyBounds.isEmpty)
+      sourceMetadata.setComponent[Bounds[K]](EmptyBounds).setComponent[Extent](Extent(0, 0, 0, 0))
```

Comment:
`Extent(0, 0, 0, 0)` would throw a construction exception: https://github.com/locationtech/geotrellis/blob/master/vector/src/main/scala/geotrellis/vector/Extent.scala#L117-L118

I guess it was not covered with tests?

Reply:
So `Extent.isEmpty` https://github.com/locationtech/geotrellis/blob/master/vector/src/main/scala/geotrellis/vector/Extent.scala#L151 would always return `true`?

Reply:
@jbouffard ah sry; i was thinking about a different thing. It's good here.